graphql_ws/base.py

```diff
@@ -144,16 +148,9 @@ def send_error(self, connection_context, op_id, error, error_type=None):
             ' GQL_CONNECTION_ERROR or GQL_ERROR'
         )
 
-        error_payload = {
-            'message': str(error)
-        }
+        error_payload = {'message': str(error)}
 
-        return self.send_message(
-            connection_context,
-            op_id,
-            error_type,
-            error_payload
-        )
+        return self.send_message(connection_context, op_id, error_type, error_payload)
 
     def unsubscribe(self, connection_context, op_id):
         if connection_context.has_operation(op_id):
```

Comment:
the `connection_context.get_operation(op_id).dispose()` seems to depend on the old `rxpy`-dependency that has been removed from `graphql-core` and is effectively removed in this branch by removing `setup_observable_extension`.

this causes an unhandled exception on `unsubscribe`